frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -69,12 +69,12 @@
   "This object has been updated.": "This object has been updated.",
   "Click reload to see the new version.": "Click reload to see the new version.",
   "Save": "Save",
+  "Click on the names to access advanced options for <0></0>.": "Click on the names to access advanced options for <0></0>.",
```

Comment:
Please do a similar change for the other language files so that we don't lose our translation ere.